app/assets/javascripts/float-kit/addon/services/menu.js

```diff
@@ -2,11 +2,11 @@ import { action } from "@ember/object";
 import { getOwner } from "@ember/owner";
 import { schedule } from "@ember/runloop";
 import Service from "@ember/service";
-import { TrackedSet } from "@ember-compat/tracked-built-ins";
+import { TrackedObject } from "@ember-compat/tracked-built-ins";
```

Comment:
All the fixes around registeredMenus and registeredTooltips were legit I think. Basically I discovered that:

```javascript
set.add(instance);
set.has(instance); // false
```